manifests/router/link_route_pattern.pp

```diff
@@ -10,7 +10,7 @@
 #
 define qpid::router::link_route_pattern(
   $prefix    = 'queue.',
-  $connector = 'broker',
+  $connector = '',
```

Comment:
Agreed undef isn't reliable. Once you accept you're testing a string, the contents of the default string become somewhat arbitrary. I'm not worried about it :)